packages/devtools_app/lib/src/performance/performance_controller.dart

```diff
@@ -208,15 +210,17 @@ class PerformanceController extends DisposableController
       await processTraceEvents(allTraceEvents);
       _processing.value = false;
 
+      _timelinePollingRateLimiter = RateLimiter(
+        timelinePollingRateLimit,
+        () async => await _pullTraceEventsFromVmTimeline(),
+      );
+
       // Poll for new timeline events.
       // We are polling here instead of listening to the timeline event stream
       // because the event stream is sending out of order and duplicate events.
       // See https://github.com/dart-lang/sdk/issues/46605.
       _pollingTimer = Timer.periodic(const Duration(seconds: 2), (_) async {
```

Comment:
any reason not to tweak this to 1 second or 500ms?

Reply:
done